src/index.ts

```diff
@@ -103,8 +101,9 @@ export async function typescriptOfSchema(
   }
 
   const tableNames = tables.map(t =>
-    normalizeName(optionsObject.transformTypeName(t)),
+    normalizeName(getTableTypeName(t, schema, !!options.prefixWithSchemaNames)),
```

Comment:
@danvk I believe transformTypeName has been used wrong in places where it's given the table name.  

I found this out when fixing the implementation of this: https://github.com/danvk/pg-to-ts/pull/14#discussion_r839645868

For some reason the code was transforming the table name to camelCase IF the camelCase option was set. From what I can see the camelCase option is for transforming the column names. I think my implementation can stay if we don't want the table names to be exported as camelCase

Reply:
Yes, that makes sense to me. Let's fix this in a future change.